README.md

```diff
@@ -14,7 +14,8 @@ You can still find the (archived) previous version of Hyrise on [Github](https:/
 
 When referencing this version of Hyrise, please use the following bibtex entry:
 <details><summary>(click to expand)</summary>
-```
+  
+```bibtex
```

Comment:
The syntax highlighting is not the actual change - the newline is. Without that, the current formatting is entirely broken.